config/locales/en.yml

```diff
@@ -32,9 +32,8 @@ en:
       invalid_postcode: "This isn't a valid postcode."
       invalid_postcode_sub: "Try checking it and entering it again."
       valid_postcode_no_match: "We couldn't find this postcode."
-      valid_postcode_no_match_sub_html: "Try entering it again or <a href='http://local.direct.gov.uk/LDGRedirect/Start.do?mode=1'>look up the council</a>."
+      valid_postcode_no_match_sub_html: "Try checking it and entering it again."
```

Comment:
We should get content team to review the words here to make sure it's an ok message.